### DISCUSSION TITLE
Black Friday dos Cursos do Willian Justen

### DISCUSSION THREAD

**Discussion by Roger-Melo** [frontendbr/forum#804] [Dicas]
Olá pessoal. 
O Willian Justen lançou uma promoção nos cursos dele e queria compartilhar com vcs. 

![captura de tela 2017-11-08 as 18 51 20](https://user-images.githubusercontent.com/29297788/32573689-ed3e3ebe-c4b5-11e7-91ff-bd47a9e8f80c.png)
![captura de tela 2017-11-08 as 18 51 30](https://user-images.githubusercontent.com/29297788/32573691-ed78b4cc-c4b5-11e7-91c4-08299e7c6581.png)
![captura de tela 2017-11-08 as 18 51 35](https://user-images.githubusercontent.com/29297788/32573692-ed9e7f9a-c4b5-11e7-89a0-d02851ce4832.png)

Quem tiver interesse: 

[Curso de JS com TDD + ES6](https://www.udemy.com/js-com-tdd-na-pratica/?couponCode=BLACKFRIDAYJA)  
[Curso de ES6](https://www.udemy.com/curso-completo-de-javascript-moderno-es6/?couponCode=BLACKFRIDAYJA)
[Curso de SVG](https://www.udemy.com/aprendendo-svg-do-inicio-ao-avancado/?couponCode=BLACKFRIDAYJA&siteID=je6NUbpObpQ-ObOBR_lebF.836Hgn214vQ&LSNPUBID=je6NUbpObpQ)

**Comment by giioohbernini** [comment]
Tenho o de svg, recomendo!!

Os cursos do mestre @willianjusten são top!

**Comment by willianjusten** [comment]
Eita, valeu pelo compartilhamento ahuahua

**Comment by devigor** [comment]
@willianjusten a promoção vai até quando??

**Comment by willianjusten** [comment]
Até acabarem os cupons xD

**Comment by grodrigues** [comment]
@willianjusten Vou comprar hoje sem falta.
